content/docs/concepts/authorization/v2.x/configuration/_index.md

```diff
@@ -68,7 +68,7 @@ A `storage` entity in Container Storage Modules Authorization consists of the st
    | vault.kvEngine            | The path to the KV secrets engine.                                                   | Yes      | secret  |
    | vault.path                | The location within the store that the credentials for the array are stored.         | Yes      | -       |
    | skipCertificateValidation | A boolean that enables/disables certificate validation of the backend storage array. | No       | true    |
-   | pollInterval              | PollInterval is the polling frequency to test the storage connectivity.              | No       | 30s     |
+   | pollInterval              | PollInterval is the polling frequency to test the storage connectivity.              | No       | 5m     |
```

Comment:
I think it should be the same as in samples. 10m in this case if we decide to keep 10 minutes

Reply:
Is this poll sync or poll array connectivity?

Reply:
Fixed to 30s.